cryptofeed/connection.py

```diff
@@ -5,19 +5,20 @@
 associated with this software.
 '''
 import asyncio
-from decimal import Decimal
+import aiohttp
+import requests
```

Comment:
im not sure why these were re-ordered. I like the imports to be in blocks of std library imports, followed by a block of external, 3rd party library imports, followed by a block of internal library imports. Please undo/correct these changes here

Reply:
It must've been my IDE since it does all the importing for me automatically. I'll keep your preferences in mind for future pull requests.